sound/soc/sof/intel/hda-stream.c

```diff
@@ -249,6 +218,11 @@ int hda_dsp_stream_trigger(struct snd_sof_dev *sdev,
 	struct hdac_stream *hstream = &stream->hstream;
 	int sd_offset = SOF_STREAM_SD_OFFSET(hstream);
 	u32 dma_start = SOF_HDA_SD_CTL_DMA_START;
+#if !IS_ENABLED(SND_SOC_SOF_HDA_ALWAYS_ENABLE_DMI_L1)
```

Comment:
@keyonjie I find this funny. you were the one who fixed the bug with the missing CONFIG_ and you missed this yourself?

Reply:
removing this patch from the series.